lib/sync-UpdatesModel_fhdb.js

```diff
@@ -0,0 +1,91 @@
+var syncUtil = require('./sync-util');
+var fhdb;
+
+var setFHDB = function(db) {
+  fhdb = db;
+};
+
+var create = function(dataset_id, data, cb) {
+  // Check if the Update has already been recorded, then create it if it doesn't
+  fhdb({
+    "act": "list",
+    "type": dataset_id + "-updates",
+    "eq": {
+      "cuid": data.cuid,
+      "type": data.type,
+      "action": data.action,
+      "hash": data.hash,
+      "uid": data.uid
+    }
+  }, function (err, found) {
+    if (err) return cb(err);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)